pkgs/development/python-modules/catppuccin-catwalk/default.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pillow,
+}:
+buildPythonPackage rec {
+  pname = "catppuccin-catwalk";
+  version = "0.4.0";
```

```suggestion
  version = "0.4.0";
  format = "pyproject";
```
Add `format`, please.